### PR TITLE
Modify descriptions on landing page

### LINKED ISSUE #136: Feature section improvement
**Description**

It would be nice to shorten the feature section - to replace the bullet points by one sentence description for each feature.   

Use description of each feature from here https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/004-overview-key-components.md

Add the bullet points description of each feature from the landing page to the relevant sections on the Docs page in the Overview. Also, link features to the relevant sections on the Docs page. 

Thank you!
Ana

### PR DESCRIPTION
**Pull request by derberg**
**Description**

Changes proposed in this pull request:

- Make shorter descriptions
- Add links to relevant components

**Related issue(s)**
Resolves #136

### CODE REVIEW

File: src/components/landing/Features.js [hunk 1]
```diff
@@ -22,19 +22,19 @@ const Features = () => {
   const sections = landingPage.features || [];
   return (
     <div className="row kyma-attr row--sm-align-center">
-      <h2 className="col-12 col-md-12">Key features</h2>
+      <h2 className="col-12 col-md-12">
+         <a href="/docs/root/kyma#overview-key-components">Key features</a>
```
Comment (pkosiec) [inline src/components/landing/Features.js]: 1. Hardcoded internal url - I'd suggest moving it to locales JSON
2. Use `Link` component from Gatsby
3. Put link as parent of `h2`

File: src/components/landing/Features.js [hunk 2]
```diff
@@ -22,19 +22,19 @@ const Features = () => {
...
+      </h2>
       {sections.map(section => {
         return (
           <section className="col-4 col-md-12" key={section.id}>
-            <h3>{section.headline}</h3>
+            <h3><a href="{section.url}">{section.headline}</a></h3>
```
Comment (pkosiec) [inline src/components/landing/Features.js]: 2. Use `Link` component from Gatsby
3. Put link as parent of `h3`

File: src/components/landing/WhatIs.js [hunk 3]
```diff
@@ -9,28 +9,17 @@ const WhatIs = () => {
   const headline = landingPage[sectionPrefix].headline;
   const paragraphs = landingPage[sectionPrefix].paragraphs || [];
 
-  const github = {
-    name: ui.socialMedia.github.name,
-    url: ui.socialMedia.github.url,
-  };
+  const docs = "/docs/root/kyma#overview-how-to-start";
```
Comment (pkosiec) [inline src/components/landing/WhatIs.js]: I'd suggest moving it to locales JSON

File: src/components/landing/WhatIs.js [hunk 4]
```diff
@@ -9,28 +9,17 @@ const WhatIs = () => {
   const headline = landingPage[sectionPrefix].headline;
```
Comment (pkosiec) [inline src/components/landing/WhatIs.js]:
```
./src/components/landing/WhatIs.jsModule Warning (from ./node_modules/eslint-loader/index.js):/Work/go/src/github.com/kyma-project/website/src/components/landing/WhatIs.js  3:8  warning  'ui' is defined but never used  no-unused-vars✖ 1 problem (0 errors, 1 warning)
```

File: src/components/landing/Features.js (hunk 1 above)
Comment (pkosiec) [reply src/components/landing/Features.js]: Please put `Key features` into localization JSON

File: src/components/landing/WhatIs.js (hunk 4 above)
Comment (derberg) [reply src/components/landing/WhatIs.js]: but you do know CI should do that part, not you ;)

**Comment by derberg** [comment]
@pkosiec all fixed, thanks for review
regarding the styling, can you be more specific? you mean the style of headlines that now as links are a bit too blue-ish?

**Comment by pkosiec** [comment]
@derberg Yes, exactly. Links style should match links from the Community section. So apart from the color, please apply also `text-decoration: none`.
 
screen shot 2018-11-29 at 08 57 13

**Comment by derberg** [comment]
style after my fixes

screen shot 2018-11-29 at 11 04 33

File: src/components/landing/WhatIs.js (hunk 4 above)
Comment (pkosiec) [reply src/components/landing/WhatIs.js]: You're right! We would love to see your contribution in `Jenkinsfile`!

Comment (derberg) [reply src/components/landing/WhatIs.js]: I don't work with such old technologies ( dinologies) 😆 
Come back to me once you use Prow

**Pull request merged**